.github/release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+  "@web3-storage/worker-utils": "0.5.1-dev"
```

Comment:
q: where does this version string come from?

Reply:
This is the current package version, as shown in `package.json`. It includes a pre-release identifier to indicate it’s still in development. 
For more info, check out the SemVer specification: https://semver.org/#spec-item-9. (I didn’t know this was possible either!)

Reply:
I see, so we need to keep the string here in sync with the one in `package.json`, is that correct? Any way that could happen automatically? What would happen if we update it in one place but forget doing so in the other?

I'm trying to evaluate if it'd be worth it to look into automating this.